variables.tf

```
@@ -120,6 +120,18 @@ variable "alb_authenticate_cognito" {
   default     = {}
 }
 
+variable "alb_enable_deletion_protection" {
+  description = "If true, deletion of the load balancer will be disabled via the AWS API. This will prevent Terraform from deleting the load balancer. Defaults to false."
+  type        = bool
+  default     = false
```

lets set these to default to `null` so that they default to what is provided upstream by the provider or by the AWS API:
```suggestion
  default     = null
```